registrations/models.py

```diff
@@ -250,6 +253,69 @@ def is_user_editable_resources(self):
         )
 
 
+class RegistrationUser(models.Model):
```

Comment:
Is this model supposed to represent an user or an participant/attendee? The name used here is a bit confusing for me at least; if I see `FooUser`, I assume it's an actual user model that's been inherited from `AbstractUser` or something 😅

Reply:
Purpose of this model is to grant read permissions to signup list of a single registration/event. RegistrationUsers are the persons who check the participants present while the event is going on. 
We use email to check that current user has permissions to read signups so make it easier for event/registration admin to grant permissions.  

I agree with you that `RegistrationUser` is quite confusing name. Do you have any suggestion for better name? 
Would something like `RegistrationAccess` make more sense? 

I was also thinking RegistrationPermissions but that's confusing as well.

Reply:
Hm... okay, so if I read the code correctly, you need to be an authenticated user to see all of the signups, right? And we're using this through the Registration model to check if a user's email matches any of the emails in `registration_users`... so isn't this basically something that could be done as a many-to-many relation between `Registration` and `User`? 😅 

As for the name, if this can't be done as a many-to-many relation instead then I suppose it acts like a read access table? So something like `RegistrationAccess` or `RegistrationUserAccess` would be more descriptive 🤔

Reply:
This would be nice to be done with many-to-many relation between `Registration` and `User`. But the problem with that approach is that the user might not exist when admin user is creating/updating `Registration`. Also, at the moment only superadmin users are allowed to get list of users, so admin user wouldn't be able to search users that he want to add to a registration.

My personal favourite for the name is `RegistrationUserAccess`?

@harriris-vincit What do you think?

Reply:
`RegistrationUserAccess` seems good enough considering also the terminology that we have agreed (should try to match the commonly known names with the ones we use in the code, and we have agreed "Registration user" as the name for this role).

Reply:
Ok, I renamed `RegistrationUser` model as `RegistrationUserAccess`. Changes are committed in https://github.com/City-of-Helsinki/linkedevents/pull/631/commits/bfc5875f9bf8b91cc78a1bb390231d39859f16b9